charts/console/README.md

```diff
@@ -205,6 +219,88 @@ config:
   license: "${ENTERPRISE_LICENSE}"
 ```    
 
+### Install with a basic SSO configuration
+
+```yaml
+config:
+  organization:
+    name: "my-org"
+
+  admin:
+    email: "admin@my-org.com"
+    password: "admin"
+
+  database:
+    host: ''
+    port: 5432
+    name: 'postgres'
+    username: ''
+    password: ''
+  sso:
+    oauth2:
+      - name: 'auth0'
+        default: true
+        client-id: <client_id>
+        client-secret: <client_secret>
+        callback-uri: http://localhost/auth/oauth/callback/auth0
+        openid:
+          issuer: https://conduktor-staging.eu.auth0.com/
+  
+  license: '<license_key>'
+```
+
+### Install with a kafka cluster
+
+```yaml
+config:
+  organization:
+    name: "my-org"
+
+  admin:
+    email: "admin@my-org.com"
+    password: "admin"
+
+  database:
+    host: ''
+    port: 5432
+    name: 'postgres'
+    username: ''
+    password: ''
+  clusters:
+    - id: my-local-kafka-cluster
+      name: My Local Kafka Cluster
+      color: '#0013E7'
+      bootstrapServers: 'my-bootstrap-server:9092'
+      schemaRegistry:
+        id: my-schema-registry
+        url: 'http://my-schema-registry:8081'
+```
+
+### Provide the license as a Kubernetes Secret
```

Comment:
can you also add or complement this section with others secrets like admin account, database access, sso, clusters, ...